### PR TITLE
Lisa taylor

### PR DESCRIPTION
**Pull request by lisaMTayl**
Found an awesome tutorial, but it needs revision.  I'll work more on it in the morning.

### CODE REVIEW

File: src/controls/Controls.spec.js [hunk 1]
```diff
@@ -1,6 +1,60 @@
 // Test away!
 import React from 'react';
-import renderer from 'react-test-renderer';
-import { render, fireEvent } from '@testing-library/react';
+import renderer from 'react-test-renderer';  // yarn add -D react-test-renderer
```
Comment (AdetunjiShennaike) [inline src/controls/Controls.spec.js:3]: the comments =']

File: src/controls/Controls.spec.js [hunk 2]
```diff
@@ -1,6 +1,60 @@
...
+    render(<Controls />);
+    expect((/lock gate/i));
+    expect((/close gate/i));
+  });
+
+  it('Should see open gate and lock buttons when gate is currently closed and unlocked', () => {
+    const { getByText } = render(<Controls locked={ false } closed={ true }/>);
+    const button = getByText(/open gate/i);
+    act(() => {fireEvent.click(button)});
+      expect(getByText(/open gate/i));
```
Comment (AdetunjiShennaike) [inline src/controls/Controls.spec.js:90]: this goes outside of act

File: src/controls/Controls.spec.js [hunk 3]
```diff
@@ -7,11 +7,74 @@ import Controls from './Controls';
 
 // add snapshot to catch and approve changes to Controls.js
 describe('<Controls />', () => {
+
+  it('should display', () => {
+    let load = render(<Dashboard/>)
+
```
Comment (AdetunjiShennaike) [inline src/controls/Controls.spec.js:13]: call the render lol or just post id as render without the let

File: src/controls/Controls.spec.js [hunk 4]
```diff
@@ -7,11 +7,74 @@ import Controls from './Controls';
...
+  })
+
   it('should match snapshot', () => {
     const tree = renderer.create(<Controls />).toJSON();
     expect(tree).toMatchSnapshot();
   });
 
+  // begin tutorial
+  //display
```
Comment (AdetunjiShennaike) [inline src/controls/Controls.spec.js:22]: do you understand my comments lol

**Review by AdetunjiShennaike — APPROVED** [review]

File: src/controls/Controls.spec.js (hunk 1 above)
Comment (lisaMTayl) [reply src/controls/Controls.spec.js:3]: I'm trying with the comments, I swear!

File: src/controls/Controls.spec.js (hunk 4 above)
Comment (lisaMTayl) [reply src/controls/Controls.spec.js:22]: ...ish  mostly

**Pull request merged**